### ISSUE TITLE
Would it possible for someone to write up some ELI5 instructions for this?

### ISSUE THREAD

**Issue by flyize** [hugobloem/esp-ha-speech#12]
I'm not even exactly sure how to flash it right now? Or maybe that means I need to wait? 😄

**Comment by hugobloem** [comment]
Hi,
Thank you for your interest! I am currently building a settings UI which should make things much easier to set up. I am not sure how long that will take. However, if it takes too long I will write some instructions for the software as of now. ☺️

**Comment by flyize** [comment]
I'm anxiously awaiting that then. Thank you!
